src/userCLI/build.gradle

```diff
@@ -42,6 +42,12 @@ distDocker {
     tagVersion = cli_docker_version
 }
 
+detekt {
+    config = files("$rootDir/config/detekt/detekt.yml")
+    buildUponDefaultConfig = true
+    baseline = file("$rootDir/config/detekt/baseline-userCLI.xml")
+}
+
```

Comment:
I don't _think_ this is necessary - it should already be possible to run `./gradlew :userCLI:detekt` at the top-level, because the `detekt` task is defined in the top-level `build.gradle` file. However, it uses a project-wide baseline. If we want to have component-level baselines then we should move the existing top-level definition into `src/app/build.gradle`, renaming `basename-main.yml` to `baseline-app.yml`. This adds to this PR rather than modifying it, so could be another ticket.